media-gfx/pencil/pencil-0.6.1.1.ebuild

```diff
@@ -40,7 +40,7 @@ src_install() {
 	einstalldocs
 
 	# install target not yet provided
-	# emake INSTALL_ROOT="${D}" install
+	# emake INSTALL_ROOT="${D}" install || die "emake install failed"
```

Comment:
emake does not require || die as the eclass handles it. So, the original comment is valid.